Makefile

```diff
@@ -91,7 +91,7 @@ compiler_spec: $(O)/compiler_spec ## Run compiler specs
 
 .PHONY: docs
 docs: ## Generate standard library documentation
-	$(BUILD_PATH) ./bin/crystal docs src/docs_main.cr $(DOCS_OPTIONS) --project-name=Crystal --project-version=$(CRYSTAL_VERSION)
+	$(BUILD_PATH) ./bin/crystal docs src/docs_main.cr $(DOCS_OPTIONS) --project-name=Crystal --project-version=$(CRYSTAL_VERSION) --source-refname=$(CRYSTAL_CONFIG_BUILD_COMMIT)
```

Comment:
if there is a fallback to the local working copy git information, why this change?

Reply:
Because it should also work if the source code is not a git working directory (for example from a tarball). Then `CRYSTAL_CONFIG_BUILD_COMMIT` can be configured externally.